scripts/o3de/o3de/repo_properties.py

```diff
@@ -115,7 +275,7 @@ def create_remote_object_archive(src_data_path: pathlib.Path,
         with zip_path.open('rb') as f:
             json_data['sha256'] = hashlib.sha256(f.read()).hexdigest()
 
-    return json_data
+        return json_data
```

Comment:
by indenting this function, is there now the possibility of the function finishing without returning anything?